pages/windows/tzutil.md

```diff
@@ -0,0 +1,15 @@
+# tzutil
+
+> A tool for displaying or configuring the system time zone.
+
+- Display the current time zone:
+
+`tzutil /g`
```

Comment:
What does `g` mean, curious

Reply:
I think `g` is for `get` :thinking:

Reply:
Maybe then "Get the current time zone" ?